pkgs/development/python-modules/pymavlink/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, future, lxml }:
+
+buildPythonPackage rec {
+  pname = "pymavlink";
+  version = "2.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0y9rz3piddzdjpp7d5w9xi6lc9v9b4p4375a5hrfiphrmhds85i3";
+  };
+
+  propagatedBuildInputs = [ future lxml ];
+
+  # No tests included in PyPI tarball
+  doCheck = false;
```

Comment:
please check out from github if the tarball doesn't contain any tests

Reply:
The PyPI tarball contains generated code for the MAVLink messages. They can be generated at build time, but this requires the message definitions found here: https://github.com/mavlink/mavlink/tree/master/message_definitions/v1.0

The PyPI tarball uses a seemingly random commit (found by manually bisecting to find the commit that matched the tarball): https://github.com/mavlink/mavlink/commit/107cf8823bdbdd68053bbd990e25885c935a48fa

I think we should use the same commit as upstream for compatibility, but I'm not sure it's worth the extra maintenance burden just so we can run the tests.

Reply:
ah, didn't realize they did some special seeding when distributing, then you can disregard :)